config/config_test.yml

```diff
@@ -3,6 +3,7 @@ imports:
 
 parameters:
     doctrine.website.mysql.password: 'VdtLtifRh4gt37T'
+    doctrine.website.mysql.host: '127.0.0.1'
```

Comment:
So `localhost` does not work here, but 127.0.0.1 does? What do the error messages look like?

Reply:
```
PHP Fatal error:  Uncaught PDOException: SQLSTATE[HY000] [2002] No such file or directory in /home/runner/work/doctrine-website/doctrine-website/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:27
Stack trace:
#0 /home/runner/work/doctrine-website/doctrine-website/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php(27): PDO->__construct('mysql:host=loca...', 'root', 'VdtLtifRh4gt37T', Array)
#1 /home/runner/work/doctrine-website/doctrine-website/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOMySql/Driver.php(25): Doctrine\DBAL\Driver\PDOConnection->__construct('mysql:host=loca...', 'root', 'VdtLtifRh4gt37T', Array)
#2 /home/runner/work/doctrine-website/doctrine-website/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(362): Doctrine\DBAL\Driver\PDOMySql\Driver->connect(Array, 'root', 'VdtLtifRh4gt37T', Array)
#3 /home/runner/work/doctrine-website/doctrine-website/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(428): Doctrine\DBAL\Connection->connect()
#4 /home/runner/work/doctrine-website in /home/runner/work/doctrine-website/doctrine-website/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php on line 93
```

Reply:
Oh ok so it thinks `localhost` is a file, so a socket?

Reply:
Yes, a socket is used with localhost. I want to prevent further configurations to the database in the github actions to keep it simple.